CHANGELOG.md

```diff
@@ -51,13 +51,26 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (version) [\#7848](https://github.com/cosmos/cosmos-sdk/pull/7848) [\#7941](https://github.com/cosmos/cosmos-sdk/pull/7941) `version --long` output now shows the list of build dependencies and replaced build dependencies.
 
 ### State Machine Breaking Changes
-
-* (x/upgrade) [\#7979](https://github.com/cosmos/cosmos-sdk/pull/7979) keeper pubkey storage serialization migration from bech32 to protobuf. 
+* (x/upgrade) [\#7979](https://github.com/cosmos/cosmos-sdk/pull/7979) keeper pubkey storage serialization migration from bech32 to protobuf.
 
 ### Bug Fixes
 
 * (crypto) [\#7966](https://github.com/cosmos/cosmos-sdk/issues/7966) `Bip44Params` `String()` function now correctly returns the absolute HD path by adding the `m/` prefix.
 
+
+### API Breaking
+
+* [\#X](https://github.com/cosmos/cosmos-sdk/pull/X) Updated the `codec.Marshaler` interface
+  * Moved `MarshalAny` and `UnmarshalAny` helper functions to `codec.Marshaler` and renamed to `MarshalIfc` and `UnmarshalIfc` respectively. These functions must take interface as a parameter (not a concrete type nor `Any` object). Underneath they use `Any` wrapping for correct protobuf serialization.
+
+
+### Improvements
+
+* [\#X](https://github.com/cosmos/cosmos-sdk/pull/X)
+  * Created `IfcJSONMarshaler` interface and `NewJSONAnyMarshaler` - a helper method to use codec.(Un)marsjalJSON withing a client context.
```

Comment:
this is not the case anymore, right?

Reply:
No, I will remove it. `IfcJSONMarshaler` is not needed anymore.